# http://developer.berlios.de/bugs/?func=detailbug&bug_id=19110&group_id=5358
#
# the files where these LDFLAGS would cause problems:
# * src/build_tools/autorevision/Makefile
# * src/sdk/Makefile
# * src/src/Makefile
#--- acinclude.m4.orig
#+++ acinclude.m4
#@@ -69,12 +69,10 @@ AC_ARG_ENABLE(debug, [AC_HELP_STRING([--enable-debug], [turn on debugging (defau
#     if test "x$enable_debug" = "xyes"; then
#         CFLAGS="-g -DDEBUG -DCB_AUTOCONF $CFLAGS"
#         CXXFLAGS="-g -DDEBUG -DCB_AUTOCONF $CXXFLAGS"
#-        LDFLAGS="-Wl,--no-undefined"
#         AC_MSG_RESULT(yes)
#     else
#         CFLAGS="-O2 -ffast-math -DCB_AUTOCONF $CFLAGS"
#         CXXFLAGS="-O2 -ffast-math -DCB_AUTOCONF $CXXFLAGS"
#-        LDFLAGS="-Wl,--no-undefined"
#         AC_MSG_RESULT(no)
#     fi
# ])
--- configure.orig
+++ configure
@@ -11932,13 +11932,11 @@ fi
     if test "x$enable_debug" = "xyes"; then
         CFLAGS="-g -DDEBUG -DCB_AUTOCONF $CFLAGS"
         CXXFLAGS="-g -DDEBUG -DCB_AUTOCONF $CXXFLAGS"
-        LDFLAGS="-Wl,--no-undefined"
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
     else
         CFLAGS="-O2 -ffast-math -DCB_AUTOCONF $CFLAGS"
         CXXFLAGS="-O2 -ffast-math -DCB_AUTOCONF $CXXFLAGS"
-        LDFLAGS="-Wl,--no-undefined"
         { $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
 $as_echo "no" >&6; }
     fi
